### ISSUE TITLE
RSS Links Broken

### ISSUE THREAD

**Issue by Smylers** [guardian/frontend#11422]
The links to RSS feeds are broken on at least this [series](http://www.theguardian.com/commentisfree/series/loose-canon) and [journalist](http://www.theguardian.com/profile/victoriacoren).

Specifically they both just link to http://www.theguardian.com/id/rss, which 404s, thereby breaking the ‘Subscribe’ button in Firefox, for example:
````
<link href="http://www.theguardian.com/id/rss" rel="alternate" type="application/rss+xml" title="Loose canon | Opinion | The Guardian RSS feed"/>
<link href="http://www.theguardian.com/id/rss" rel="alternate" type="application/rss+xml" title="Victoria Coren Mitchell | The Guardian RSS feed"/>
````

They should link to the page that they are on but with `/rss` on the end; those URLs exist ([series](http://www.theguardian.com/commentisfree/series/loose-canon/rss) and [journalist](http://www.theguardian.com/profile/victoriacoren/rss)), and are of feeds with content that matches the labels in the `title` attributes above, so presumably what was intended.

Specifically I think changing the link to simply use `href="./rss"` should work (and be fewer bytes), leaving the browser to resolve it against the current URL.

**Comment by beltsonata** [comment]
Is this going to be fixed? They're broken everywhere I've looked.

**Comment by Smylers** [comment]
Fixed by @johnduffell in #12446 — thanks.
